.github/workflows/bench.yml

```diff
@@ -12,6 +12,9 @@ jobs:
         os: [ubuntu-latest]
 
     steps:
+      # Cancel queued workflows from earlier commits in this branch
+      - uses: fkirc/skip-duplicate-actions@master
+
```

Comment:
Maybe this indentation breaks the actions?

Reply:
Mmm it shouldn't afaik, yaml lets you do it

Reply:
It looks like inconsistent indentation does not work in YAML. The following lines have less indentation.